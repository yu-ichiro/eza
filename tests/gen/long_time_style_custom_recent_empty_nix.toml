bin.name = "eza"
args = "tests/timestamp_test_dir --long --no-permissions --no-filesize --no-user --sort=modified --time-style='+NON_RECENT\n\n'"
status.code = 2
