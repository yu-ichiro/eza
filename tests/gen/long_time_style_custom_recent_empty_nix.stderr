error: Custom timestamp format for recent files is empty, please supply a chrono format string at the second line.
